kptncook/api.py

```diff
@@ -28,11 +28,9 @@ class KptnCookClient:
     Client for the kptncook api.
     """
 
-    def __init__(
-        self, base_url=settings.kptncook_api_url, api_key=settings.kptncook_api_key
-    ):
+    def __init__(self, base_url=settings.kptncook_api_url, api_key=settings.kptncook_api_key):
         self.base_url = base_url
-        self.headers = {"content-type": "application/json"}
+        self.headers = {"content-type": "application/json", "Accept": "application/vnd.kptncook.mobile-v8+json", "User-Agent": "Platform/Android/12.0.1 App/7.10.1", "hasIngredients": "yes"}
```

Comment:
Oh, this looks super interesting. Is it possible to get more information out of kptncook?
Have to try this..

Reply:
Yeah, I've seen it [here](https://github.com/EGo4/syncKptncookWithTandor/blob/main/fetchkptncook/KptnCookCommunicationService.cs#L79) and decompiled the app to verify.